src/test_l1_hdmi_cec_driver.c

```diff
@@ -1296,40 +1292,35 @@ void test_hdmicec_hal_l1_setRxCallback_positive( void )
     /* Positive result */
     result = HdmiCecOpen (&handle);
     //if init is failed no need to proceed further
-    UT_ASSERT_EQUAL ( result, HDMI_CEC_IO_SUCCESS );
+    UT_ASSERT_EQUAL_FATAL ( result, HDMI_CEC_IO_SUCCESS );
 
     logicalAddress = DEFAULT_LOGICAL_ADDRESS;
 
     //Set logical address for TV
     //logicalAddress = 0;
     result = HdmiCecAddLogicalAddress(handle, logicalAddress);
-    if (HDMI_CEC_IO_SUCCESS != result) {
-        /*Cleanup before exiting */
-        result = HdmiCecClose (handle);
-        if (HDMI_CEC_IO_SUCCESS != result) { UT_FAIL ("HdmiCecClose failed"); }
-        UT_FAIL ("HdmiCecAddLogicalAddress failed");
-    }
+    if (HDMI_CEC_IO_SUCCESS != result) { HDMICEC_TERM(); UT_FAIL_FATAL ("HdmiCecAddLogicalAddress failed"); }
 
     //Get logical address for STB
     result = HdmiCecGetLogicalAddress(handle, &logicalAddress);
-    if (HDMI_CEC_IO_SUCCESS != result) { HDMICEC_ASSERT_AUTO_TERM(); UT_FAIL ("HdmiCecGetLogicalAddress failed"); }
+    if (HDMI_CEC_IO_SUCCESS != result) { HDMICEC_TERM(); UT_FAIL_FATAL ("HdmiCecGetLogicalAddress failed"); }
 
     //Using NULL callback
     result = HdmiCecSetRxCallback(0, DriverReceiveCallback, 0);
-    if (HDMI_CEC_IO_INVALID_ARGUMENT != result) { HDMICEC_ASSERT_AUTO_TERM(); UT_FAIL ("HdmiCecSetRxCallback failed"); }
+    if (HDMI_CEC_IO_INVALID_ARGUMENT != result) { HDMICEC_TERM(); UT_FAIL_FATAL ("HdmiCecSetRxCallback failed"); }
```

Comment:
testing invalid param in positive test case
 also do we need calls HdmiCecAddLogicalAddress and HdmiCecGetLogicalAddress?